sdk/java/src/main/java/com/gojek/feast/RequestUtil.java

```diff
@@ -53,29 +46,19 @@
    * Parse a feature reference proto builder from the given featureRefString
    *
    * @param featureRefString string feature reference to parse from.
-   * @param ignoreProject If true, would ignore if project is specified in given ref string.
-   *     Otherwise, throwws a {@link IllegalArgumentException} if project is specified.
    * @return a parsed {@link FeatureReference.Builder}
    */
-  public static FeatureReference.Builder parseFeatureRef(
-      String featureRefString, boolean ignoreProject) {
+  public static FeatureReference.Builder parseFeatureRef(String featureRefString) {
     featureRefString = featureRefString.trim();
     if (featureRefString.isEmpty()) {
       throw new IllegalArgumentException("Cannot parse a empty feature reference");
     }
-    FeatureReference.Builder featureRef = FeatureReference.newBuilder();
-
-    // parse project if specified
     if (featureRefString.contains("/")) {
-      if (ignoreProject) {
-        String[] projectSplit = featureRefString.split("/");
-        featureRefString = projectSplit[1];
-      } else {
-        throw new IllegalArgumentException(
-            String.format("Unsupported feature reference: %s", featureRefString));
-      }
+      throw new IllegalArgumentException(
+          String.format("Unsupported feature reference: %s", featureRefString));
```

Comment:
Can we be more specific with this exception?

Reply:
Corrected.